tests/benchmarks/power_consumption/qdec/boards/nrf54l15pdk_nrf54l15_cpuapp.overlay

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright 2024 Nordic Semiconductor ASA
+ * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
```

Comment:
actually I just noticed, this is wrong, this should be nrf54l15dk, not pdf

Reply:
https://github.com/nrfconnect/sdk-nrf/commit/479a68b6ce108599e601d858526ba5044849de5e this commit needs adding

Reply:
It will be fixed in yet another PR/backport :smile:
https://github.com/nrfconnect/sdk-nrf/pull/18453/files#diff-81222de48522dddb92ea8c9866df530a507b589474e81de3f12af73cd43999c2

Reply:
I think I need this PR to be merged first.

Then, re run backport job on PR with fix:
https://github.com/nrfconnect/sdk-nrf/actions/runs/11702277129
https://github.com/nrfconnect/sdk-nrf/pull/18453